client/src/api/requests/signUp.ts

```diff
@@ -0,0 +1,10 @@
+import { post } from 'src/api/api'
+import { SignUpFormServerData } from 'src/components/SignUpForm/SignUpForm'
```

Comment:
why do u write types by yourself, they generate automatically
![Uploading image.png…]()

Reply:
All fields are optional, for live validation. Now it's not needed